utils/clients/reputation/client.go

```diff
@@ -62,7 +62,7 @@ func (iro *IsDrainReputableOpts) GenerateQueryString() (url.Values, error) {
 // IsDrainReputableResponse is what the reputation server
 // will send back when we ask if a wallet is reputable
 type IsDrainReputableResponse struct {
-	Cohort        int    `json:"cohort"`
+	Cohorts       []int  `json:"cohorts"`
 	Justification string `json:"justification"`
```

Comment:
Since the Justification field is not being used anywhere and with multiple cohorts we'd have multiple justifications, I removed it. Only the `cohorts` field is returned by reputation now.

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/1350/commits/1592adacbd49470e63773cd658f21308c2c3af1e